src/UWP/UWP.csproj

```diff
@@ -145,7 +145,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Acr.UserDialogs">
-      <Version>6.3.10</Version>
+      <Version>6.5.1</Version>
```

Comment:
We can't only update UWP here. We should update all projects. I'll look into that and report back

Reply:
Ok, I don't actually need the update for UWP to work, so I'll update this PR dropping this patch.